allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_lims_api.py

```diff
@@ -80,7 +80,7 @@ def get_ophys_session_id(self) -> int:
         id used to initialize the API"""
         query = """
                 SELECT os.id FROM ophys_sessions os
-                JOIN ophys_experiment oe ON oe.ophys_session_id = os.id
+                JOIN ophys_experiments oe ON oe.ophys_session_id = os.id
```

Comment:
how did this never come up before?

Reply:
The BehaviorOphysSession didn't try to pull the ophys_session_id prior to this PR :/